srcpkgs/discord-ptb/template

```diff
@@ -12,16 +12,16 @@ maintainer="0x5c <dev@0x5c.io>"
 license="custom:Proprietary"
 homepage="https://discord.com/"
 distfiles="https://dl-ptb.discordapp.net/apps/linux/${version}/discord-ptb-${version}.tar.gz"
-checksum=77bf0d9d06773242e37bc987ac8ea7a07da20f6a04bd22777039ec9b342c5187
-_license_checksum=03580ea1297c38f31c4437555f2dbf9dae25a91c9e61391d7141cf8cc036aeff
+checksum=7333a5e3d176b20da0e7da739877bd1d8f3d3a6aad71efe5f954ec90ff534db5
+_license_checksum=df4707f9ed6889fc8c64661ef054daeddd97ba1c1da8c0dedf22f57c925398d7
 nopie=yes
 restricted=yes
 repository=nonfree
 
 post_extract() {
 	$XBPS_FETCH_CMD -o eula https://discord.com/terms; cat eula |
 		w3m -dump -I utf-8 -T text/html |
-		sed -n '/Discord Terms of Service/,/^Imagine a place$/p' > EULA
+		sed -n '/Discord.s Terms of Service/,/^IMAGINE A PLACE$/p' > EULA
```

Comment:
Testing this via the below command, the sed command does not end where I believe you intend it to -- "IMAGINE A PLACE" should have bee left unchanged.

`curl https://discord.com/terms | w3m -dump -I utf-8 -T text/html | sed -n '/Discord.s Terms of Service/,/^IMAGINE A PLACE$/p'`

Reply:
The sed cut has to end at the start of the footer, which is the catchphrase. That catchphrase used to be in lowercase, which they changed since the last version to the uppercase

Reply:
I thought I had checked specifically for that, but apparently not, thanks for catching this